firmware/application/ui_navigation.cpp

```diff
@@ -554,6 +559,30 @@ void SystemStatusView::rtc_battery_workaround() {
     }
 }
 
+void SystemStatusView::new_sdcard_structure_checker() {
```

Comment:
Following a firmware upgrade, I still feel that we should provide an easy one-click YES/NO modal that automates movement of the pre-existing user files and directories to the new folder locations when we detect the old folder layout.  The warning code below is helpful, but I think we could do better.

Reply:
> I still feel that we should provide an easy one-click YES/NO modal that automates movement of the pre-existing user files and directories to the new folder locations

That would be a good idea. I think I can move CAPTURES. About other dir, since it could contain mixed both user and official things, I'm not sure how to handle that?

Reply:
Auto remove implemented